src/v2/BaseLoginRouter.js

```diff
@@ -153,7 +153,25 @@ export default Router.extend({
   },
 
   handleIdxResponseFailure (error = {}) {
-    // 1. loosely check whether is IDX error response
+    // special case: `useInteractionCodeFlow` is true but the Org is not using OIE
+    const featureMsg = 'The requested feature is not enabled in this environment.';
+    if (error?.error === 'access_denied' && error.error_description === featureMsg) {
+      // simulate an IDX error response
+      error = Object.assign({
+        details: {
+          messages: {
```

Comment:
I think this should work if you add message like this instead 
```
“messages”: {
    “type”: “array”,
    “value”: [
      {
        “message”: “Any message",
        “i18n”: {
          “key”: “error.feature.disabled”
        },
        “class”: “ERROR”
      }
    ]
  } (
```

Reply:
OK, so we can use the raw error as message and then provide an i18n key for translation. Should I provide a test for error when i18n key is not set?